### PR TITLE
Measure embedded database performance - Closes #5261

### LINKED ISSUE #5261: Measure embedded database performance
### Description
With the introduction of EmbeddedDB, we need to measure the performance of `EmbeddedDB`.
Following  `EmbeddedDB` to consider for performance measure `LevelDB` and `RocksDB`.

### Motivation
- Optimized database storage size
- Better read/write and batch performance
- Ease of import/export of data

### Acceptance Criteria
- Measure the database storage size
- Random read
- Random write
- BatchWrite

### Additional Information
- Use 15kb, 50kb, 100kb and 150kb payload for measuring the read/write/batch operation
- Use digital ocean droplet with configuration
```
Ubuntu: 18.04.3 (LTS) x64
8 GB / 4 CPUs
160 GB SSD disk
```

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #5261 

### How was it solved?

- Created scripts to benchmark `leveldb` and `rocksdb`

### How was it tested?

- Benchmarked using command `node benchmark/index.js 15000`
- Used `15Kb, 50Kb, 100Kb, 150Kb`
- Key -> String, Value -> Buffer
- Used ubuntu with configuration `Ubuntu: 18.04.3 (LTS) x64
8 GB / 4 CPUs
160 GB SSD disk`

### CODE REVIEW

**Comment by ManuGowda** [comment]
15Kb
```
LevelUP: del(string):Buffer x 15000(bytes) x 51,503 ops/sec ±18.72% (26 runs sampled)
LevelUP: put(string):Buffer x 15000(bytes) x 22,668 ops/sec ±19.58% (26 runs sampled)
LevelUP: get(string):Buffer x 15000(bytes) x 61,274 ops/sec ±13.94% (32 runs sampled)
LevelUP: batch(string):Buffer x 15000(bytes) x 2,427 ops/sec ±11.34% (44 runs sampled)
RocksDB: get(string):Buffer x 15000(bytes) x 47,785 ops/sec ±14.31% (28 runs sampled)
Fastest is LevelUP: get(string):Buffer x 15000(bytes)
RocksDB: del(string):Buffer x 15000(bytes) x 51,415 ops/sec ±21.31% (39 runs sampled)
Fastest is LevelUP: del(string):Buffer x 15000(bytes),RocksDB: del(string):Buffer x 15000(bytes)
RocksDB: batch(string):Buffer x 15000(bytes) x 105,386 ops/sec ±66.37% (19 runs sampled)
Fastest is RocksDB: batch(string):Buffer x 15000(bytes)
RocksDB: put(string):Buffer x 15000(bytes) x 32,666 ops/sec ±13.56% (43 runs sampled)
Fastest is RocksDB: put(string):Buffer x 15000(bytes)
```

**Comment by ManuGowda** [comment]
50Kb
```
LevelUP: del(string):Buffer x 50000(bytes) x 50,609 ops/sec ±25.45% (14 runs sampled)
LevelUP: put(string):Buffer x 50000(bytes) x 14,355 ops/sec ±20.95% (34 runs sampled)
LevelUP: get(string):Buffer x 50000(bytes) x 50,070 ops/sec ±19.63% (32 runs sampled)
RocksDB: del(string):Buffer x 50000(bytes) x 39,479 ops/sec ±32.27% (8 runs sampled)
Fastest is LevelUP: del(string):Buffer x 50000(bytes),RocksDB: del(string):Buffer x 50000(bytes)
LevelUP: batch(string):Buffer x 50000(bytes) x 674 ops/sec ±14.39% (34 runs sampled)
RocksDB: put(string):Buffer x 50000(bytes) x 4,483 ops/sec ±24.78% (16 runs sampled)
Fastest is LevelUP: put(string):Buffer x 50000(bytes)
RocksDB: get(string):Buffer x 50000(bytes) x 46,941 ops/sec ±29.65% (29 runs sampled)
Fastest is LevelUP: get(string):Buffer x 50000(bytes),RocksDB: get(string):Buffer x 50000(bytes)
RocksDB: batch(string):Buffer x 50000(bytes) x 133,690 ops/sec ±12.28% (36 runs sampled)
Fastest is RocksDB: batch(string):Buffer x 50000(bytes)
```

**Comment by ManuGowda** [comment]
100Kb
```
LevelUP: put(string):Buffer x 100000(bytes) x 5,446 ops/sec ±19.04% (16 runs sampled)
LevelUP: del(string):Buffer x 100000(bytes) x 53,184 ops/sec ±31.21% (14 runs sampled)
LevelUP: get(string):Buffer x 100000(bytes) x 41,040 ops/sec ±20.37% (15 runs sampled)
RocksDB: put(string):Buffer x 100000(bytes) x 11,592 ops/sec ±16.66% (17 runs sampled)
Fastest is RocksDB: put(string):Buffer x 100000(bytes)
RocksDB: get(string):Buffer x 100000(bytes) x 48,913 ops/sec ±14.79% (24 runs sampled)
Fastest is RocksDB: get(string):Buffer x 100000(bytes),LevelUP: get(string):Buffer x 100000(bytes)
RocksDB: del(string):Buffer x 100000(bytes) x 48,948 ops/sec ±10.19% (28 runs sampled)
Fastest is RocksDB: del(string):Buffer x 100000(bytes),LevelUP: del(string):Buffer x 100000(bytes)
LevelUP: batch(string):Buffer x 100000(bytes) x 679 ops/sec ±5.71% (51 runs sampled)
RocksDB: batch(string):Buffer x 100000(bytes) x 146,248 ops/sec ±20.69% (7 runs sampled)
Fastest is RocksDB: batch(string):Buffer x 100000(bytes)
```

**Comment by ManuGowda** [comment]
150Kb
```
LevelUP: put(string):Buffer x 150000(bytes) x 5,508 ops/sec ±22.79% (17 runs sampled)
LevelUP: batch(string):Buffer x 150000(bytes) x 389 ops/sec ±10.68% (26 runs sampled)
LevelUP: get(string):Buffer x 150000(bytes) x 44,966 ops/sec ±19.13% (29 runs sampled)
LevelUP: del(string):Buffer x 150000(bytes) x 70,292 ops/sec ±13.37% (37 runs sampled)
RocksDB: get(string):Buffer x 150000(bytes) x 39,282 ops/sec ±15.83% (17 runs sampled)
RocksDB: batch(string):Buffer x 150000(bytes) x 81,421 ops/sec ±23.89% (27 runs sampled)
RocksDB: put(string):Buffer x 150000(bytes) x 8,674 ops/sec ±10.63% (35 runs sampled)
RocksDB: del(string):Buffer x 150000(bytes) x 38,684 ops/sec ±19.42% (17 runs sampled)
Fastest is LevelUP: get(string):Buffer x 150000(bytes)
Fastest is RocksDB: batch(string):Buffer x 150000(bytes)
Fastest is RocksDB: put(string):Buffer x 150000(bytes)
Fastest is LevelUP: del(string):Buffer x 150000(bytes)
```

**Comment by nazarhussain** [comment]
@ManuGowda Would you please add some descriptions for each benchmark or some explanations. 

> put(string):Buffer

As above names are not really helpful to understand. Does above means you used string to save or buffer? Or used buffer and converted It to string and saved?

File: elements/lisk-db/README.md [hunk 1]
```diff
@@ -8,6 +8,67 @@
...
+|   batch   |  x 389 ops/sec ±10.68%   | x 81,421 ops/sec ±23.89% | Rocksdb |  20830   |
+
+Benchmark summary for 15, 50, 100 and 150 Kb
+
+| Payload(Kilobyte) | Winner  |
+| :---------------: | :-----: |
+|        15         | Rocksdb |
+|        50         | Leveldb |
+|        100        | Rocksdb |
+|        150        | Leveldb |
```
Comment (nazarhussain) [inline elements/lisk-db/README.md:70]: How you calculated winners for this category?

**Pull request merged**